### PR TITLE
fix: [CDS-36997]:  Deployment type selection intermittent issue fixed

### PR DESCRIPTION
**Pull request by bhavya-sinha**
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by bhavya-sinha** [comment]
retrigger titlecheck

**Comment by bhavya-sinha** [comment]
retrigger titlecheck

**Comment by bhavya-sinha** [comment]
retrigger titlecheck

**Review by kapilkumawat86 — APPROVED** [review]

**Review by kapilkumawat86 — DISMISSED** [review]

**Comment by bhavya-sinha** [comment]
trigger cypress

File: src/modules/75-cd/components/PipelineSteps/ServerlessAwsLambdaServiceSpec/__tests__/__snapshots__/ServerlessAwsLambdaServiceSpec.test.tsx.snap [hunk 1]
```diff
@@ -5,91 +5,6 @@ exports[`ServerlessAwsLambdaServiceSpec tests When stepViewType is Edit render p
   <div
     class="serviceDefinition"
   >
-    <div
```
Comment (chetanvekariya-harness) [inline src/modules/75-cd/components/PipelineSteps/ServerlessAwsLambdaServiceSpec/__tests__/__snapshots__/ServerlessAwsLambdaServiceSpec.test.tsx.snap:8]: Could you pass deploymentType for this test case. This change should not appear, I think.

**Review by chetanvekariya-harness — CHANGES_REQUESTED** [review]

**Review by chetanvekariya-harness — APPROVED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Pull request merged**